nodejs/user-interface/modals/new-task.js

```diff
@@ -22,21 +20,11 @@ module.exports = (prefilledTitle) => {
       Blocks.Input({ label: 'New task', blockId: 'taskTitle' }).element(
         textInput(prefilledTitle),
       ),
-      Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser', optional: true }).element(
+      Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser' }).element(
         Elements.UserSelect({
           actionId: 'taskAssignUser',
-        }),
-        //.initialUser(User.)
+        }).initialUser(currentUser),
       ),
-      //OR
-      // Instead of a select user element we use a conversations element so that we can set filter out bot users something like:
-      // Blocks.Input({ label: 'Assign user', blockId: 'taskAssignUser', optional: true }).element(
-      //   Elements.ConversationSelect({
-      //     actionId: 'taskAssignUser',
-      //   }) 
-      //   .filter(['im'])
-      //   .excludeBotUsers(true) //breaks app
-      // ),
```

Comment:
I opted against the conversations select because I wasn't 100% what the payload looks like. It may well be the better choice.